parser/src/symbols.rs

```diff
@@ -24,13 +24,19 @@ pub mod predefined {
     pub const Sum: Symbol = Symbol::new(3);
     /// The symbol `prod`
     pub const Prod: Symbol = Symbol::new(4);
+    /// The symbol `program_hash`
+    pub const ProgramHash: Symbol = Symbol::new(5);
+    /// The symbol `stack_inputs`
+    pub const StackInputs: Symbol = Symbol::new(6);
```

Comment:
These things are specific to MidenVM and shouldn't be part of AirScript. They are examples of specific public inputs. I don't remember seeing this before, so hopefully I didn't miss it, but if they're anywhere else (other than tests), they should be removed.

Reply:
I've gone ahead and made the `Symbol` implementation of `Ord` use the underlying string value, if we ever get into optimization in the future, we can revisit this then.